src/lib-components/Flexible/MediaGallery/BannerImage.vue

```diff
@@ -72,9 +77,18 @@ const classes = computed(() => {
         />
       </div>
       <MediaBadge v-if="nItems > 1" :is-expanded="expanded">
-        {{ nItems }}
-        images
-        <svg class="glyph-expand">
+        <span v-if="theme">
```

Comment:
same comment as before, can we specify theme.value === ftva? Or add a class to this space and use display:none to hide it on the default.scss theme file